src/js/utils/JobUtil.js

```diff
@@ -0,0 +1,90 @@
+import Job from '../structs/Job';
+
+const getMatchingProperties = function (job, item) {
```

Comment:
Could we get a description of this function or some comments? Not sure what exactly is going on here.
Picking out properties from a schema, but what is the `getter`?